src/middlewared/middlewared/etc_files/local/nginx/nginx.conf.mako

```diff
@@ -56,6 +57,10 @@
             })
     ip6_list = [f'[{ip}]' for ip in ip6_list]
 
+    wg_config = middleware.call_sync('datastore.config', 'system.truecommand')
```

Comment:
why datastore and not truecommand.config?

Reply:
Because we don't expose `wg_address` in `truecommand.config`

Reply:
Why would it be in datastore and not it .config?